index.ipynb

```diff
@@ -0,0 +1,283 @@
+{
```

Comment:
<ul><li>These plots are really busy and hard to read. Maybe for the first set you could just pick a bright mag and a faint mag instead of all magnitudes. For the last two maybe a 2x2 grid separating out each corner sensor into its own plot?</li><li>Include a couple sentence explaining exactly how you accomplished the averaging vs stacking analysis. I think you told us in person that you created single donut stamps from stacks to get the stacked results right? </li><li>Also it would be interesting to see a sample from one of the experiments of the single stacked Zernike estimate, the average Zernike estimate and the range of individual Zernike estimates that go into the average. </li><li>Furthermore based upon the above, could you comment based upon that plot requested on how robust each method is to a single detected donut with a bad pixel or other feature that may skew the Zernike estimate based upon that stamp?</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-sitcom/sitcomtn-085/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
<ul><li>chosen for the first set (comparing stacking vs averaging for different corner sensors with no background) only mag 12 (brightest) and 17 (faintest), to show that for no background there's little difference between the two strategies (and removed the previous ones that showed a variety of magnitudes per sensor).</li></ul>

Reply:
<ul><li>split the PSF FWHM plot into a 2x2 panel plot with one panel per corner sensor </li></ul>

Reply:
<ul><li>added a plot that illustrates the variance of Zernikes that went into the average</li></ul>

Reply:
<ul><li>added a plot summarizing standard deviation of individual Zernike mode estimates</li></ul><p> </p>

Reply:
<ul><li>added a plot illustrating how donuts arrays were stacked within each group </li></ul>